### PR TITLE
Add minor text changes to monthly statistics page

### PR DESCRIPTION
**Pull request by vassyz**
[Monthly statistics: Final content changes](https://trello.com/c/7Q49Kggf/993-monthly-statistics-final-content-changes)

Messed up the branch name, ignore that.

### CODE REVIEW

**Comment by vassyz** [comment]
[Monthly statistics: Final content changes](https://trello.com/c/7Q49Kggf/993-monthly-statistics-final-content-changes)

**Review by JR-G — APPROVED** [review]

File: app/views/publications/monthly_statistics/v2/show.html.erb [hunk 1]
```diff
@@ -150,7 +149,7 @@
...
+    <p class="govuk-body">Every recruitment cycle starts on the second Tuesday of October and not on the same date each calendar year. The figures for ‘last recruitment cycle’ and the current cycle in the table are calculated by counting the same number of days from when applications opened. This means days in last cycle will not be the same calendar date as days in this cycle. For example, day one of last cycle was on <%= CycleTimetable.find_opens(CycleTimetable.previous_year).to_fs(:govuk_date) %>, but day one of the current cycle was <%= CycleTimetable.apply_opens.to_fs(:govuk_date) %>.</p>
```
Comment (MaeveRoseveare) [inline app/views/publications/monthly_statistics/v2/show.html.erb:152]: The second date (<%= CycleTimetable.apply_opens.to_fs(:govuk_date) %>) should be the date Find opened this cycle (3 October 2023).

Comment (MaeveRoseveare) [inline app/views/publications/monthly_statistics/v2/show.html.erb:152]:
```suggestion
    <p class="govuk-body">Each recruitment cycle does not start on the same date each calendar year. The figures for 'last cycle' include data up to and including the date which was the same number of days from the first day of the previous cycle as the publication date is from the first day of the current cycle. This means days in last cycle will not be the same calendar date as days in this cycle. For example, day one of last cycle was on <%= CycleTimetable.find_opens(CycleTimetable.previous_year).to_fs(:govuk_date) %>, but day one of the current cycle was <%= CycleTimetable.apply_opens.to_fs(:govuk_date) %>.</p>
```

**Review by MaeveRoseveare — CHANGES_REQUESTED** [review]
Looks good, thank you. I've left comments from Steven where things should be changed and these are 2 extra ones:

1. Make 'application' plural under 'All applications rejected'

Screenshot 2023-11-24 at 15 53 29

2. Remove line about EEA people:

Screenshot 2023-11-24 at 15 54 03

File: config/locales/publications/itt_monthly_report_generator.yml [hunk 2]
```diff
@@ -59,13 +59,13 @@ en:
             last_cycle: number_of_candidates_accepted_to_same_date_previous_cycle
         rejected:
           title: All applications rejected
-          summary: Candidates whose applications this cycle have all been rejected, excluding applications which were withdrawn. Candidates with one or more application this cycle that has not been rejected are excluded.
+          summary: Candidates whose applications this cycle have all been rejected, excluding applications which were withdrawn. Candidates with one or more applications this cycle that has not been rejected are excluded.
```
Comment (MaeveRoseveare) [inline config/locales/publications/itt_monthly_report_generator.yml:62]:
```suggestion
          summary: Candidates whose applications this cycle have all been rejected, excluding applications which were withdrawn. Candidates with one or more applications this cycle that have not been rejected are excluded.
```

**Review by MaeveRoseveare — APPROVED** [review]
Looks good. Just one 'has' change to 'have' under 'All applications rejected'. But you should be able to just commit that directly. Then you can put a curse on me.

**Pull request merged**